src/qt/guiutil.h

```diff
@@ -218,6 +218,7 @@ namespace GUIUtil
     QString formatNiceTimeOffset(qint64 secs);
 
     QString formatBytes(uint64_t bytes);
+    QString formatBps(uint64_t bytes);
```

Comment:
Should add doxygen documentation

see: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments

Reply:
hmmm. well, the units indicate that it is a different metric, so I'm not sure if the addition of the word "total" is needed. What do other people think?

Reply:
@jarolrod I don't think doxygen docs exist for formatBytes, so perhaps that can be done in another pull request to do it for a number of functions.

Reply:
@rebroad 
Since you're adding this function (`formatBps`) you should add the Doxygen comment with it. `formatBytes` not including a Doxygen when it was introduced is another issue that can be addressed in another PR.

Reply:
@jarolrod given that formatBps is so similar to formatBytes, I'm averse to doing the Doxygen for formatBytes as part of this pull, and I'm also averse to doing Doxygen for just formatBps without formatBytes in this pull, but I'm fine with doing a later pull request to add the Doxygen for both formatBytes and formatBps.